include/SDL_video.h

```diff
@@ -1509,6 +1510,18 @@ extern DECLSPEC int SDLCALL SDL_SetWindowHitTest(SDL_Window * window,
                                                  SDL_HitTest callback,
                                                  void *callback_data);
 
+/**
+ * Request a window to give a signal, e.g. a visual signal, to demand attention from the user.
+ *
+ * \returns a number >= 1 or a negative error code on failure; call
```

Comment:
It actually returns 0 on success or a negative error code on failure

Reply:
Ah, i copied from another function and read it more like (returns a number >= 1 or a negative error code) on failure :D